pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/phimanifest/PhiManifestService.java

```diff
@@ -209,15 +211,23 @@ private boolean hasParticipantConsentedToSharedLearning(ElasticSearchParticipant
         }
         String dateOfBirth = participant.getDsm().get().getDateOfBirth();
         String dateOfMajority = (String) participant.getDsm().get().getDateOfMajority();
-        if (StringUtils.isBlank(dateOfMajority) || DateTimeUtil.isAdult(dateOfMajority)) {
-            return (Boolean)getAdultParticipantConsentedToTumorAnswer(participant, ddpInstanceDto.getStudyGuid())
+        String participantStatus = participant.getStatus().orElse("");
+        if (StringUtils.isBlank(dateOfMajority) || participant.hasCompletedActivity(CONSENT_ACTIVITY_CODE)) {
+            //self adult enrollment
+            return (Boolean) getAdultParticipantConsentedToTumorAnswer(participant, ddpInstanceDto.getStudyGuid())
                     .orElse(false);
+        } else {
+            //pediatric or aged-up in lost to followup
+            return isPediatricValidForPHI(participant, dateOfBirth, participantStatus);
         }
+    }
+
+    private static boolean isPediatricValidForPHI(ElasticSearchParticipantDto participant, String dateOfBirth, String participantStatus) {
         int age = DateTimeUtil.calculateAgeInYears(dateOfBirth);
         boolean hasCompletedPediatricConsentAddendum = participant.hasCompletedActivity(CONSENT_ADDENDUM_PEDIATRICS_ACTIVITY_CODE);
 
         if (hasCompletedPediatricConsentAddendum) {
-            if (age >= 7) {
+            if (age >= 7 || participantStatus.equalsIgnoreCase(CONSENT_SUSPENDED)) {
```

Comment:
agedup participants should fall into this >7 age for sure any way but explicitly checking for CONSENT_SUSPENDED will help just incase DOB isn't in ES DSM document. 
Doesnt hurt to include in OR , hence added